README.md

```diff
@@ -1,6 +1,6 @@
 # :fire: Furnace
 
-> Melts down the [GOLD](https://github.com/govau/gold) components into a zip file.
+> Melts down the [GOLD](https://github.com/govau/designsystem) components into a zip file.
```

Comment:
You also wanna change the text in the link from `GOLD` to `design system`

Reply:
Yeah I'll do that